File: week-07/day-2/Workshop 16.5.2023/takes_longer.py
# When saving this quote a disk error has occured. Please fix it!
# Insert the words "always takes longer than" between the words "It" and "you"!

quote = "Hofstadter's Law: It you expect, even when you take into account Hofstadter's Law."
add = "always takes longer than "
new_quote = quote[:21] + add + quote[21:]
quote=new_quote
print(quote)

File: week-02/day-4/Workshop 13.4.2023/src/TakesLonger.java
public class TakesLonger {
    public static void main(String[] args) {
        String quote = "Hofstadter's Law: It you expect, even when you take into account Hofstadter's Law.";

        // When saving this quote a disk error has occurred. Please fix it!
        // Insert the words "always takes longer than" between the words "It" and "you" using the StringBuilder class!
        System.out.println(quote.indexOf("y"));
        buldingString(quote);

    }
    public static void buldingString (String inp) {
        StringBuilder bs = new StringBuilder(inp);
        bs.insert(20, " always takes longer than");
        System.out.println(bs);


    }
}
